kwc-blockly-flyout.js

```diff
@@ -301,6 +301,8 @@ class KwcBlocklyFlyout extends PolymerElement {
         this._listeners = this._listeners || [];
         this._listeners.push(Blockly.bindEvent_(root, 'mousedown', null, this._blockMouseDown(block)));
         this._listeners.push(Blockly.bindEvent_(rect, 'mousedown', null, this._blockMouseDown(block)));
+        this._listeners.push(Blockly.bindEvent_(root, 'mouseup', null, this._blockMouseUp(block)));
```

Comment:
The mouseup part of the gesture is handled by Blockly and this will lead to two listeners to be added and having a race between them. We can probably change the behaviour of `doBlockClick` in `core/blockly/gesture.js` to prevent the creation of a block for all flyouts

Reply:
Ok.

Reply:
Resolved.

Reply:
You need to import the new file `gesture.js` from `import.js`